charts/openklant/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.6.3 (2025-03-31)
+
+- Adding "extra deploy" possibility to deploy additional resources like config maps, deployments, etc. The variable `extraDeploy` is added to take the configuration.
```

Comment:
Usually it is good practice to put the issue number if there is one :slightly_smiling_face: 
```suggestion
- [#201] Adding "extra deploy" possibility to deploy additional resources like config maps, deployments, etc. The variable `extraDeploy` is added to take the configuration.
```